app/controllers/application_controller.rb

```diff
@@ -172,13 +172,13 @@ def resource_base
   # this method is used with nested resources, where obj_id is passed into the parameters hash.
   # it automatically updates the search text box with the relevant relationship
   # e.g. /hosts/fqdn/reports # would add host = fqdn to the search bar
-  def setup_search_options
+  def setup_search_options(with_and_operator: false)
     @original_search_parameter = params[:search]
     params[:search] ||= ""
     params.keys.each do |param|
       if param =~ /(\w+)_id$/
         if params[param].present?
-          query = "#{Regexp.last_match(1)} = #{params[param]}"
+          query = "#{with_and_operator ? ' and ' : ''}#{Regexp.last_match(1)} = #{params[param]}"
           params[:search] += query unless params[:search].include? query
```

Comment:
As per Ewoud's suggestion, we can do this instead:
```suggestion
          unless params[:search].include? query
            params[:search] += ' and ' unless params[:search].blank?
            params[:search] += query
          end
```